### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Zearin**
Minor formatting tweaks

### CODE REVIEW

**Comment by nfischer** [comment]
@Zearin Docs changes need to be done in `src/*.js` files first, then update the README using `npm run gendocs`

File: README.md [hunk 1]
```diff
@@ -640,11 +640,11 @@ touch({ '-r': FILE }, '/path/to/some/dir/source.js');
 ```
 
 Update the access and modification times of each FILE to the current time.
-A FILE argument that does not exist is created empty, unless -c is supplied.
-This is a partial implementation of *[touch(1)](http://linux.die.net/man/1/touch)*.
+A FILE argument that does not exist is created empty, unless `-c` is supplied.
+This is a partial implementation of *[`touch(1)`](http://linux.die.net/man/1/touch)*.
```
Comment (nfischer) [inline README.md]: If we're using backtick formatting, let's remove the italics

File: README.md [hunk 2]
```diff
@@ -273,10 +273,10 @@ Arguments:
 
 Display the list of currently remembered directories. Returns an array of paths in the stack, or a single path if +N or -N was specified.
 
-See also: pushd, popd
+See also: `pushd`, `popd`
 
 
-### echo([options,] string [, string ...])
+### `echo([options,] string [, string ...])`
```
Comment (nfischer) [inline README.md]: I think it looks a bit better with the headers as-is. Were there any formatting bugs I'm overlooking?

File: README.md [hunk 3]
```diff
@@ -400,11 +400,11 @@ ln('file', 'newlink');
 ln('-sf', 'file', 'existing');
 ```
 
-Links source to dest. Use -f to force the link, should dest already exist.
+Links source to dest. Use `-f` to force the link, should `dest` already exist.
```
Comment (nfischer) [inline README.md]: Also wrap `source` and `dest` in backticks

**Comment by Zearin** [comment]
Whoops!  That’s what I get for not looking more carefully.  I’ll update the docs in src as you said.

File: README.md (hunk 2 above)
Comment (Zearin) [reply README.md]: No.  I just prefer the monospace font for any documentation that is referencing a command/option/option-value.  

But I can omit this when I redo the Pull Request.  :)

**Comment by Zearin** [comment]
Reopening as a new Pull Request…

**Comment by nfischer** [comment]
You can always just add more commits to the same branch 😄

**Pull request closed without merging**